src/stories/Avatar.stories.tsx

```diff
@@ -1,13 +1,14 @@
 import React from 'react';
-import { ComponentStory, ComponentMeta } from '@storybook/react';
+import { StoryFn, Meta } from '@storybook/react';
 import { Avatar } from '../components/Avatar/Avatar';
+import { AvatarProps } from '../interfaces/CommonInterface';
 
 export default {
     title: 'ReactComponentLibrary/Avatar',
     component: Avatar
-} as ComponentMeta<typeof Avatar>;
+} as Meta<typeof Avatar>;
 
-const Template: ComponentStory<typeof Avatar> = (args) => <Avatar {...args} />;
+const Template: StoryFn<AvatarProps> = (args: AvatarProps) => <Avatar {...args} />;
```

Comment:
what is StoryFn

Reply:
IN Storybook 6.0 onward "StoryFn" refers to a new way of defining stories introduced in newer versions of Storybook.

Reply:
ok